src/components/fields/radio-button/radio-button.tsx

```diff
@@ -52,6 +53,12 @@ export const RadioButtonGroup = (props: IGenericFieldProps<IRadioButtonGroupSche
 		value ? onChange({ target: { value } }) : onChange(event);
 	};
 
+	const handleClick = (event: React.MouseEvent<HTMLButtonElement>, value?: string): void => {
+		console.log(event, value);
+		value ?? setStateValue(value);
+		value ? onChange({ target: { value } }) : onChange(event);
+	};
+
```

Comment:
- can combine `handleChange` and `handleClick` since they are doing the same thing
- remove errant `console.log`
- I think I missed this in the last PR: can make the `value` argument mandatory? so there is no need to check if value exists

Reply:
`value` can be undefined according to typing used in `StyledRadioButton` :  `InputHTMLAttributes` so can't be made mandatory in callback.

Reply:
yeah I am thinking we can simplify `handleChangeOrClick()` because I don't see much use of the event object and since the toggles and image buttons aren't giving a meaningful event object, we can remove it

so it can be
```
const handleChangeOrClick = (value: string): void => {
	setStateValue(value);
	onChange({ target: { value } });
};
```

with this, we can remove the checks and have all three radio button variants pass the value only `handleChangeOrClick(option.value)`

Reply:
Fixed.